contracts/seed/Seed.sol

```diff
@@ -242,13 +239,14 @@ contract Seed {
         require(tokenLocks[msg.sender].fundingAmount > 0, "Seed: zero funding amount");
         Lock storage tokenLock = tokenLocks[msg.sender];
         uint256 amount = tokenLock.fundingAmount;
-        seedRemainder = seedRemainder.add(tokenLock.seedAmount);
+        uint256 feeAmount = (tokenLock.seedAmount).mul(uint256(PPM)).mul(fee).div(PPM100);
+        uint256 fundingFeeAmount = (amount.mul(uint256(PPM))).mul(fee).div(PPM100);
+        seedRemainder = seedRemainder.add(tokenLock.seedAmount).add(feeAmount);
         tokenLock.seedAmount = 0;
```

Comment:
Actually we need to test this as well, I am not sure about memory model in Solidity, but I think that when we are creating new variable(which is `tokenLock` here) we are adding new memory field, and we are overwrighting not the dictionary, but the data in our variable somewhere in another place in our memory.

Reply:
Actually we are already testing it, so don't mind my comment here